clients/web/src/templates/body/systemConfiguration.jade

```diff
@@ -67,7 +67,9 @@ form.g-settings-form(role="form")
               placeholder="Default: #{defaults['core.upload_minimum_chunk_size'] || ''}",
               title="For large files, the minimum size of all but the last chunk.")
           | CORS (Cross-origin resource sharing) allows access from other
-          | websites.
+          | websites. These settings are described in detail
+          a(href="http://girder.readthedocs.org/en/latest/security.html#cors-cross-origin-resource-sharing"
```

Comment:
My real desire is that this would go to `girder.readthedocs.org/en/(version)` where `(version)` is latest if this is a build that is after the latest version, and is otherwise the appropriate `v1.2.4` or other string.  I don't see how to do that conveniently (and that would be a whole different issue).

LGTM.